app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource
+
+    def index
+      @comments = grouped_comments(accessible_ordered_comments)
+      @unread_comments = grouped_comments(accessible_ordered_comments.find_since_last_login(current_user))
+      @posted_comments = grouped_comments(accessible_ordered_comments.find_comments_by_user(current_user))
+    end
+
+    private
+
+    def accessible_ordered_comments
+      Comment.accessible_by(current_ability).joins('INNER JOIN events ON commentable_id = events.id').order('events.title', 'comments.created_at DESC')
+    end
+
+    def grouped_comments(remarks)
```

Comment:
Related of INNER JOIN, if there is any other elegant method to ask database for comments sorted by event title and date of creation, I will be happy to know it and to change my query. Our discussion about this subject ended with you telling  that I should rather do the query in view, instead of controller. 
But I believe that the logic of code can stay in controller in this case, not in view, and I preferred to create 2 private methods, one for the query into data base, the other one for grouping the comments by conference and by events. I assumed it also that the code will be more readable, compared with the case when we are asking everything in a single chained  call.
Of course, if you prefer any other way to do this, please tell me and I'll do it.